shogun-lib/src/main/java/de/terrestris/shogun/lib/security/access/BasePermissionEvaluator.java

```diff
@@ -45,13 +46,21 @@ public class BasePermissionEvaluator implements PermissionEvaluator {
     @Autowired
     private UserProviderService userProviderService;
 
+    @Autowired
+    private PublicEntityService publicEntityService;
+
+
     @Override
     public boolean hasPermission(Authentication authentication, Object targetDomainObject,
             Object permissionObject) {
 
         log.trace("About to evaluate permission for user '{}' targetDomainObject '{}' " +
             "and permissionObject '{}'", authentication, targetDomainObject, permissionObject);
 
+        if (hasPublicPermission(targetDomainObject)) {
+            return true;
+        }
```

Comment:
Why is this needed? Isn't this check included in the `BaseEntityPermissionEvaluator`?